ROADMAP.md

```diff
@@ -0,0 +1,90 @@
+
+Ratify serves several major scenarios. It seamlessly integrates with OPA Gatekeeper as the Kubernetes policy controller. By doing so, it safeguards your Kubernetes cluster against untrustworthy and non-compliant container images. Ratify functions as an external data provider for Gatekeeper, supplying artifact verification results that align with defined policies. Furthermore, Ratify can be employed at container runtime via the plugin interface, enabling policy evaluation based on its artifact verification outcomes. This enables more granular artifact verification at the Kubernetes node level. Last but not least, Ratify can be integrated into your CI/CD pipeline to ensure the trustworthiness and compliance of container images before using them. 
+
+This document presents the roadmap of Ratify that translates our strategy into practical steps.
+
+## Milestones
+
+The Ratify roadmap is divided into milestones, each with a set of features (high level) and timeline. The milestones marked as `Tentative` are subject to change based on the project’s priorities and the community’s feedback. We will prioritize releases for security or urgent fixes, so the roadmap may be adjusted and new features may be postponed to the next milestone. Any dates and features listed below in a given milestone are subject to change. See the [GitHub milestones](https://github.com/deislabs/ratify/milestones?state=open) for the most up-to-date issues and their status. We are targeting to release a new Ratify version every 3 or 4 months.
+
+### v1.0
+
+**Status**: Completed
+
+**Released date**: Sep 27, 2023
+
+**Release link**: [v1.0.0 Release Notes](https://github.com/deislabs/ratify/releases/tag/v1.0.0)
+
+**Major features**
+
+- Ratify as an external Data Provider for Gatekeeper
+- Plugin framework for extensibility
+- Policies for Notary Project signatures verification at admission control in kubernetes
+- Policies for Cosign keyless verification at admission control in kubernetes
+- High Availability support in Kubernetes (Experimental)
+
+### v1.1
+
+**Status**: Completed
+
+**Release date**: Dec 12, 2023
+
+**Release link**: [v1.1.0 Release Notes](https://github.com/deislabs/ratify/releases/tag/v1.1.0)
+
+**Major features**
+
+- Policies for assessing vulnerability reports at admission control in kubernetes
+- Policies for assessing software license at admission control in kubernetes
+- New diagnostic logs
+
+### v1.2
```

Comment:
For ongoing and future milestones, would it be better to add corresponding issue links (if any) so that people are not familiar with Ratify can be easily get the context and details?

Reply:
In this document, I aim to provide a high-level overview of major features as described in the first paragraph under `Milestone` section. I also took references of roadmap files from other communities, a few of them added concrete links. Users can view the individual milestone to check the up-to-date issues for details. There are some considerations: Some features lack tracked issues, especially those in the future without detailed requirements. Consequently, our feature list varies—some are with issue link and comprehensive, while others may not have issues or very simply described. Additionally, some features may be tracked by multiple issues or by new issues over time, I want to minimize frequent updates to this file and ensure accuracy.

Reply:
Thanks for the suggestion @FeynmanZhou . If we don't have link to the specific items, maybe we can provide a milestone link for customers who wish to see whats in the backlog. At the moment, it is difficult for customer to see whats being planed for the current release. E.g. we can could this link to v1.2 , https://github.com/deislabs/ratify/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.2.0

Reply:
@susanshi updated. Please take a look. I added links for v1.2.0 and v1.3.0, for 1.4.0 and 2.0 as they are tentative, we can update it later once it becomes clear.

Reply:
LGTM